android/src/test/java/com/revenuecat/purchases/common/ConfiguringUnitTests.kt

```diff
@@ -7,20 +7,22 @@ import io.mockk.every
 import io.mockk.mockk
 import io.mockk.mockkObject
 import io.mockk.verify
-import org.spekframework.spek2.Spek
-import org.spekframework.spek2.style.specification.describe
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
 
-object ConfiguringUnitTests : Spek({
-    describe("when setting up Purchases") {
+internal class ConfiguringUnitTests {
+    @Nested
+    @DisplayName("when setting up Purchases")
+    inner class ParsingInvalidPurchasesPeriod {
```

Comment:
i might just be missing something, but does this class name still make sense? i'm not seeing us parsing purchases periods in this class?

Reply:
no, it doesn't make sense. Fried Brain Fridays haha